spec/friendly_shipping/services/ups_json_spec.rb

```diff
@@ -520,6 +521,23 @@
       end
     end
 
+    context "if the address city name is spelled wrong", vcr: { cassette_name: "ups_json/labels/city_name" } do
+      let(:destination) do
+        FactoryBot.build(
+          :physical_location,
+          address1: "123 Disney World Dr",
+          city: 'Orlanod',
+          region: 'FL',
+          zip: '32821'
+        )
+      end
+
+      it "returns a failure with a good error message" do
+        is_expected.to be_failure
+        expect(subject.failure.to_s).to eq('{"code"=>"120802", "message"=>"Address Validation Error on ShipTo address"}')
```

Comment:
Shouldn't this just be "Address Validation Error on ShipTo address" instead of being a JSON string with the code, etc? I thought this was being pulled from the `Errordescription` header which in the VCR cassette is just the plain message string, not a JSON string.

Reply:
Yes! I was integration testing and we invoke it a little differently than the specs.